config/packages/easy_admin/entities/job_offer.yml

```diff
@@ -0,0 +1,16 @@
+easy_admin:
+    entities:
+        JobOffer:
+            class: App\Entity\JobOffer
+            list:
+                fields: ['id', 'title', 'company']
+            form:
+                fields:
+                    - { type: 'group', label: 'Delivery Details', icon: 'paperclip' }
```

Comment:
Copy and paste detected. Should be "Job Offer Information".

Reply:
Or "Job Offer Details", whatever keeping consistency around the application.

Reply:
got me. will fix